sgm/inference/api.py

```diff
@@ -55,14 +57,14 @@ class Thresholder(str, Enum):
 class SamplingParams:
     width: int = 1024
     height: int = 1024
-    steps: int = 50
-    sampler: Sampler = Sampler.DPMPP2M
+    steps: int = 40
+    sampler: Sampler = Sampler.EULER_EDM
     discretization: Discretization = Discretization.LEGACY_DDPM
     guider: Guider = Guider.VANILLA
     thresholder: Thresholder = Thresholder.NONE
-    scale: float = 6.0
-    aesthetic_score: float = 5.0
-    negative_aesthetic_score: float = 5.0
+    scale: float = 5.0
+    aesthetic_score: float = 6.0
+    negative_aesthetic_score: float = 2.5

```

Comment:
What's the reason for changing some of these constants?

Reply:
The aesthetic_score ones were plain wrong. The rest are adjustments to match streamlit defaults, which were recently tuned based on user preference testing.

Reply:
I feel like those would belong in a different PR that explained that - otherwise they'll end up in a presumably squashed PR with the title "Helpers + Streamlit refactor (102)".

Reply:
I added a doc comment that hopefully does a better job of explaining why they are what they are, and that they'll certainly change in time.